src/components/FilterBar.jsx

```diff
@@ -32,15 +33,18 @@ export default function FilterBar(props) {
   return (
     <Grid container spacing={2} className={classes.root}>
       <Grid item className={classes.alignText}>
-        <FilterDropDown label="Major" items={majors} />
-        <GradeLevelFilter />
+        {experiments.expshowMajorFilter && (
```

Comment:
```suggestion
        {experiments.expShowMajorFilter && (
```

Reply:
You can see this doesn't work by going to https://s4us-pr-569.onrender.com/scholarships?expShowMajorFilter=1&expShowGradeFilter=1.

If you check the logs the experiment is enabled but this isn't showing up.